src/components/tabs/tab.tsx

```diff
@@ -17,25 +16,21 @@ export interface EuiTabProps extends CommonProps {
 type EuiTabPropsForAnchor = EuiTabProps &
   AnchorHTMLAttributes<HTMLAnchorElement> & {
     href?: string;
-    onClick?: MouseEventHandler<HTMLButtonElement>;
-    buttonRef?: Ref<HTMLAnchorElement>;
```

Comment:
I think this was a leftover from a copy+paste. The component didn't use `ref` before, so we can just delete this part.